extensions/ipynb/package.json

```diff
@@ -9,8 +9,7 @@
 		"vscode": "^1.57.0"
 	},
 	"enabledApiProposals": [
-		"notebookEditor",
-		"notebookEditorEdit"
+		"notebookEditor"
```

Comment:
Isn't `notebookWorkspaceEdit` missing here?

Reply:
Yes I'll add it back. We obviously don't enforce this proposal :(